lib/realtime/nodes.ex

```diff
@@ -27,11 +27,6 @@ defmodule Realtime.Nodes do
   def platform_region_translator(nil), do: nil
 
   def platform_region_translator(tenant_region) when is_binary(tenant_region) do
-    platform = Application.get_env(:realtime, :platform)
-    region_mapping(platform, tenant_region)
-  end
-
-  defp region_mapping(:aws, tenant_region) do
```

Comment:
Should this be configured through the ENV vars as well? (Not in this PR but in the future?)

Reply:
yes it will be just wanted to make it in a different PR but it will be similar to the existing one 👀